code-challenges.js

```diff
@@ -21,29 +21,133 @@ const temp1 = 42
+// boiling point is 212 degrees Farenheit
+//I should expect 42 is below boiling point, 350 is above boiling point & 212 is at boiling point
+//within the function i would write and else if statement that determine is below boiling point is 42, else if above pointing is 350 and else if 212 is at boiling.
+// when I tried every number it gave the correct response
+// I received help with this question/challenge
+
+const boilingPoint = (temp) => {
+if(temp == 212){
+    return "${temp} is below boiling Point"
+} else if (temp>212){
+    return "${temp} is above the boiling point"
+}else if (temp< 212) {
+    return "${temp} is below the boiling point"
+}
+}
+
+console.log(boilingPoint(350))
+
+
+
+
+
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the variables provided below. Expected output: 10
 
+
+// Pseudo Coding
+// create a code that will combine two arrays using .concat()
+// ensure the length is returned by adding .length to another variable when console
+// logging.
+
+
 const myNumbers1 = [3, 7, 0, 36, -9]
 const myNumbers2 = [8, -7, 42, 9, 13]
 
+let twoArrays = myNumbers1.concat(myNumbers2)
+
+console.log(twoArrays.length)
+
+
+
+
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below. Expected output: "1202 ovarB"
 
-const currentCohort = "Bravo 2022"
+// Psuedo Coding
+// Create a code that will reverse the letters of a string
+// I decleared I will reverse my variable of bravoCohort declared as "Bravo 2022"
+// In order to do that I will first split that will read each letter speratle.
+// Then I will make them reverse in an array by removing the ""
+// Then I will join them with the .join "" back to a string
+// It returned reversed as expected.
+
+
+const bravoCohort = "Bravo 2022"
+
+let reverse = bravoCohort.split("").reverse().join("")
+
+console.log(reverse)
 
 
 // --------------------4) Create a statement that will evaluate whether each number in an array is even or odd. Use the test variable provided below. Expected output: "odd" "even" "odd" "even" "odd" "even"
 
+// const myArray = [13, 34, 5, 10, 27, 42]
+
+// Psuedo Coding
+// Create a statement aka loop
+// The statment must evaluate which numbers in the array above classified as an even or odd value using the % 2
+// Use i++ so it will read all numbers continously in the array
+// The expected output should be in the order of "odd" "even" "odd" "even" "odd" "even"
+// After my first try"
+// I had to change my == to === to strictly equal my outcome
+// I had to make sure I read myArray.length to get a number
+// Following the past examples, help and trial I recived the correct outcome.
+
 const myArray = [13, 34, 5, 10, 27, 42]
 
+for (let i=0; i< myArray.length; i++){
+    if (myArray[i] %2 === 0){
+        console.log ("even") ;
+    }else if (myArray[i] %2 === 1){
+        console.log ("odd");
+    }
+}
```

Comment:
Looks great!  My only suggestion is to have an else statement to catch anything that is not a number.